smart-contracts/osmosis/contracts/cl-vault/tests/test-tube/autocompound.rs

```diff
@@ -134,9 +130,8 @@ fn test_autocompound_with_rewards_swap_non_vault_funds() {
                     .unwrap()
                     .u128(),
                 expected_refund0,
-                &deposit_ratio_approx
+                APPROX_EQ_FACTOR
```

Comment:
fine for now, but it would have been better to do that in a separate PR. Here it is not necessary and added another 8 files to the PR